cmd/ctr/commands/images/push.go

```diff
@@ -57,32 +57,32 @@ var pushCommand = cli.Command{
 	creating the associated configuration, and creating the manifest
 	which references those resources.
 `,
-	Flags: append(commands.RegistryFlags, cli.StringFlag{
+	Flags: append(commands.RegistryFlags, &cli.StringFlag{
 		Name:  "manifest",
 		Usage: "Digest of manifest",
-	}, cli.StringFlag{
+	}, &cli.StringFlag{
 		Name:  "manifest-type",
 		Usage: "Media type of manifest digest",
 		Value: ocispec.MediaTypeImageManifest,
-	}, cli.StringSliceFlag{
+	}, &cli.StringSliceFlag{
 		Name:  "platform",
 		Usage: "Push content from a specific platform",
-		Value: &cli.StringSlice{},
-	}, cli.IntFlag{
+		Value: cli.NewStringSlice(""),
```

Comment:
Shouldn't this have zero arguments rather than an empty string

Reply:
Good catch, I was pulling from the migrate guide and didn't see the blank vs empty string.